packaging/hudi-spark-bundle/pom.xml

```diff
@@ -94,8 +94,6 @@
                   <include>org.apache.hive:hive-service-rpc</include>
                   <include>org.apache.hive:hive-metastore</include>
                   <include>org.apache.hive:hive-jdbc</include>
-
-                  <include>com.databricks:spark-avro_2.11</include>
```

Comment:
if we bundled `org.apache.spark:spark-avro` would n't that make life simpler for everyone?

Reply:
I can give it a shot, but we need to carefully understand the consequences of shading a spark library, inside a Jar which is being run on Spark. I remember earlier we had some issue on EMR, but don't have the exact details. Nevertheless, let me try and see if tests pass.

Reply:
I assume the tests will pass.. but I realize what you are saying..  the user could be running on a higher spark version say and we would be bundling 2.4.4 . Lets just open a JIRA to tackle this usability issue and keep it as -is now.. We can document the need for `--packages ...` when using spark-submit or spark-shell clearly for now.. and move on.

Reply:
Yeah that can be one of the problems. Created a JIRA for this issue: https://issues.apache.org/jira/browse/HUDI-516

About documentation of `--packages` are you guys going to take care of that ?